docs/customization/configuration.md

```diff
@@ -78,7 +78,7 @@ values via a `docker-compose.override.yml` or `.env` file. You can copy
  set this to a highly random string in case that changes in the future.
 - `ONI_SOLR_URL` (default = `http://solr:8983`): Solr server base URL
 - `ONI_STORAGE_PATH` (default = `(ONI base dir path)/data`): Path to batch storage
-- `SOLR_CLOUD` (default = False): Set to True if running solr in cloud mode
+- `ONI_SOLR_CLOUD` (default = 0): Set to 1 if running solr in cloud mode
```

Comment:
One last thing, would you please alphabetize this before `ONI_SOLR_URL`

Reply:
fixed.